README.md

```diff
@@ -42,7 +42,6 @@ You should file a [Pull Request](https://github.com/AlexsLemonade/OpenPBTA-analy
 We are using continuous integration software applied to the supplied test datasets to confirm that the analysis can be carried out successfully within the Docker container.
 
 ## How to Obtain OpenPBTA Data
```

Comment:
It looks like this works, but I think there is supposed to be a line after the section headers
```suggestion
## How to Obtain OpenPBTA Data

```